lib/cfn-nag/custom_rules/EC2NetworkAclMissingVpcIdRule.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class EC2NetworkAclMissingVpcIdRule < BaseRule
```

Comment:
let's drop this one - if we get better at doing syntax checks, i'd want it in the model, not as a rule